tests/functional/oauth_webchannel.js

```diff
@@ -96,6 +96,9 @@ define([
 
     'signin a verified account using an oauth app': function () {
       var self = this;
+      var options = {
```

Comment:
Can you just inline all of these `var options = {...}` declarations?

e.g., line 114 would be:

```js
.then(testIsBrowserNotifiedOfLogin(self, { shouldCloseTab: true }))
```